2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -42,4 +70,4 @@ const myBooks = [
 
 const ulElement = createBookList(myBooks);
 
-// document.querySelector('#bookList').appendChild(ulElement);
+document.querySelector('#bookList').appendChild(ulElement);
```

Comment:
Perfect! The code reads very nicely. All I'd suggest is to add some styling, but that's not the point of the assignment here :)